from core.errors.Application import ApplicationException


class RouterException(ApplicationException):
    pass
